README.adoc

```diff
@@ -40,34 +61,58 @@ Our objective is to know if the Jenkins project published a new stable version,
 [cols="2a,2a"]
 |===
 |```
-## updatecli.yaml
-sources:
+## updatecli.yamltitle: Update Jenkins Version
+
+scms:
   default:
+    kind: github
+    spec:
+      user: olblak
+      email: me@olblak.com
+      owner: "olblak"
+      repository: "charts"
+      token: mySecretTokenWhichShouldNeverUsedThisWay
+      username: olblak
+      branch: "main"
+
+sources:
+  jenkins:
+    name: Get latest Jenkins version
     kind: jenkins
     spec:
       release: weekly
+
 conditions:
   docker:
-    name: "Docker Image Published on Registry"
+    name: "Test if Docker Image Published on Registry"
+    sourceID: jenkins
     kind: dockerImage
     spec:
       image: "jenkins/jenkins"
```

Comment:
```suggestion
      image: "jenkinsci/jenkins"
```

Reply:
I am reverting this, the official docker image `jenkins/jenkins`
https://hub.docker.com/r/jenkins/jenkins/
https://hub.docker.com/r/jenkinsci/jenkins/ is deprecated

Reply:
My bad, sorry, the habit :P